src/form-elements/FormElementNumber.tsx

```diff
@@ -34,10 +35,15 @@ function FormElementNumber({
   const ariaDescribedby = useElementAriaDescribedby(id, element)
   const isPageVisible = useIsPageVisible()
 
-  const text = React.useMemo(
-    () => (typeof value === 'number' ? value.toString() : ''),
-    [value],
-  )
+  const text = React.useMemo(() => {
+    if (typeof value !== 'number') {
+      return ''
+    }
+    if (element.displayAsCurrency) {
+      return localisationService.formatCurrency(value)
+    }
+    return value.toString()
```

Comment:
Should we be using `localisationService` here to format the number?

Reply:
worth checking with Blake. I don't think so because it will add commas and current enjoyers of the number element might get their nose out of joint.

Reply:
Im happy with that, worth noting that we have made that change to the PDF though.

Reply:
ah yeah we (I) did too, shoot.